amethyst_assets/tests/prefab_test.rs

```diff
@@ -2,10 +2,10 @@ use std::time::{Duration, Instant};
 
 use amethyst_assets::{
     prefab::{register_component_type, Prefab},
-    AssetHandle, AssetStorage, DefaultLoader, Handle, LoadStatus, Loader, LoaderBundle,
+    AssetHandle, DefaultLoader, Handle, LoadStatus, Loader, LoaderBundle,
 };
 use amethyst_core::ecs::{
-    world::ComponentError, Dispatcher, DispatcherBuilder, Entity, IntoQuery, Resources, World,
+    world::ComponentError, Dispatcher, DispatcherBuilder, Entity, Resources, World,
 };
```

Comment:
☝️

Reply:
I only manually touched files under `amethyst_config`. I must have missed that during either rebase or my fights with "rusty hook". 🤷🏼‍♂️